libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -488,6 +553,11 @@ void AP_Airspeed::Log_Airspeed()
         };
         AP::logger().WriteBlock(&pkt, sizeof(pkt));
     }
+    // log _EAS_state
+    AP::logger().Write("EAS", "TimeUS,EAS,integ,vdot", "Qfff", AP_HAL::micros64(),
```

Comment:
This should probably just move into the `log_AIRSPEED` packet. It would be faster in all cases and smaller logs in the general case (one sensor), and in the two sensor case it's only 6 bytes more overhead.